cmd/influx/auth.go

```diff
@@ -21,9 +23,33 @@ func newAuthCommand() cli.Command {
 	}
 }
 
+func helpText(perm string) struct{ readHelp, writeHelp string } {
+	var helpOverrides = map[string]struct{ readHelp, writeHelp string }{
+		"user":      {"perform read actions against organization users", "perform mutative actions against organization users"},
+		"buckets":   {"perform mutative actions against organization buckets", "perform mutative actions against organization buckets"},
+		"telegrafs": {"read telegraf configs", "create telegraf configs"},
+		"orgs":      {"read organizations", "create organizations"},
+		"dbrps":     {"read database retention policy mappings", "create database retention policy mappings"},
```

Comment:
Why are these ones `create` instead of `perform mutative actions`?

Reply:
The default is `create or update` - see line 40 below.

For these I am just preserving the old help text, since I didn't have an opinion on the 'correct' help text.